### ISSUE TITLE
Pods-resources.sh fails to run momc for xcdatamodeld 

### ISSUE THREAD

**Issue by broccolii** [CocoaPods/CocoaPods#6541]
this shell script will use `UNLOCALIZED_RESOURCES_FOLDER_PATH`, but I printed `UNLOCALIZED_RESOURCES_FOLDER_PATH` in the library and found no output anything
```
xcrun momc "$RESOURCE_PATH" "${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/`basename "$RESOURCE_PATH" .xcdatamodeld`.momd"
```

I add shell script in Build Phasess like this 
```
echo "broccoli log -- BUILT_PRODUCTS_DIR ${BUILT_PRODUCTS_DIR}"
echo "broccoli log -- UNLOCALIZED_RESOURCES_FOLDER_PATH ${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
echo "broccoli log -- Path: ${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/"
```

log something

```

broccoli log -- BUILT_PRODUCTS_DIR /Users/Broccoli/Library/Developer/Xcode/DerivedData/YZCashier-fnvwglcmgjjqwihdcggahyrfsmbq/Build/Products/Debug-iphonesimulator
broccoli log -- UNLOCALIZED_RESOURCES_FOLDER_PATH 
broccoli log -- Path: /Users/Broccoli/Library/Developer/Xcode/DerivedData/YZCashier-fnvwglcmgjjqwihdcggahyrfsmbq/Build/Products/Debug-iphonesimulator//
```

How should I do?

**Comment by benasher44** [comment]
Hi @broccolii! Thanks for the report! Can you please post a sample project? Thanks!

**Comment by broccolii** [comment]
@benasher44 I commit a  [sample porject](https://github.com/broccolii/SampleProjectForPodsIssue),This is not a hundred percent, you need to try several times.

**Comment by benasher44** [comment]
Thanks @broccolii for posting the sample project! Unfortunately, I'm unable to run `pod install` in your demo project. When I do that, I get the following message:
> [!] Unable to add a source with url `http://gitlab.qima-inc.com/AppLib/RenRenPodspecs.git` named `qima-inc-applib-renrenpodspecs`.

It appears that I am unable to access this git repo

**Comment by broccolii** [comment]
You do not need to run `pod install`, only need to build this project on it

**Comment by benasher44** [comment]
Hm. I was able to get an error, but I'm not sure why exactly it's happening. For me, I get an error saying that it can't remove a previously existing `momd` file/folder. On master, there have been fixes made to how core data models are handled by CocoaPods, and in this case it should be fixed with the latest CocoaPods. We should be releasing a 1.2.1.rc.1 build this week. I'm going to close this, but please re-open if you find that this isn't fixed in 1.2.1.rc.1

**Comment by benasher44** [comment]
Thanks again for the report!
